doc/developer-documentation.md

```diff
@@ -170,6 +170,35 @@ Here, `node` refers to the name of API PubSub channel where node events are publ
 
 After following these steps, run your pipeline instance to activate your newly added test configuration.
 
+### Example to enable a new kselftest
```

Comment:
I'd suggest to make it generic:
```suggestion
### A complete example of enabling a new job
```

Reply:
Changing it to "A example of enabling a new job"